Python/Age Calculator.py

```diff
@@ -0,0 +1,4 @@
+name = input('Enter your name: ')
+birth_year = input('Enter the year you were born: ')
+age = 2019 - int(birth_year)
```

Comment:
@Cap10Kirk You are using a literal value `2019` which will make the script invalid in the coming years. Instead, you can use the current datetime from the system. [Help](https://stackoverflow.com/questions/17053099/how-to-print-current-date-on-python3) is available here.

Reply:
I have updated the code using the datetime module.